doc/7/getting-started/react/standalone/index.md

```diff
@@ -0,0 +1,121 @@
+- **Running Kuzzle V2 Stack** ([instructions here](/core/2/guides/getting-started/running-kuzzle))
+
+## Prepare your environment
+
+Create your React app and install all the dependencies from the command line using npm:
+```bash
+npx create-react-app kuzzle-playground
+```
+Install Kuzzle's Javascript SDK: 
+```bash
+cd kuzzle-playground
+npm install kuzzle-sdk@7
+```
+
+We'll rewrite the src/App.js so you can remove everything inside.
+
+## Instantiating Kuzzle SDK
+
+We have to connect the server so that our client can interact with it.
+To do this, we have to create a *src/services/kuzzle.js* file to put our kuzzle instance declaration:
+
+<<< ./snippets/kuzzle.js.snippet[js]
+
+You can now edit the *src/App.js* file to connect to Kuzzle. To do this, import the kuzzle service file and the following imports in the same time:
+
+<<< ./snippets/App.js.snippet:1[js]
+
+Now, add the app class and add in the constructor a `message` property in the state (we'll use it to store the user input message) a `username` property, `messages` property (we’ll use it to store all messages) and a `validate` property (Value that will change the display):
+
+<<< ./snippets/App.js.snippet:2[js]
+
+Then we will establish the connection to kuzzle and create the index and collection of our chat if they don't exist. Add the following `valid()` method in the App Component:
+
+<<< ./snippets/App.js.snippet:3[js]
+
+## Get the username
+
+First of all, we need to know the user's name. Let's start by adding the following function.
+
+<<< ./snippets/App.js.snippet:4[js]
+
+As you can see we'll need the `handleInputChange()` function to save the `username` variable
+
+<<< ./snippets/App.js.snippet:5[js]
+
+## Display the messages
+
+Then, create the following functions to fetch and display the messages:
+
+<<< ./snippets/App.js.snippet:6[js]
+
+The function `fetchMessage()` will [search](/sdk/js/7/controllers/document/search) for the first hundred newest messages and store them in our array, before subscribing to changes in the messages collection. We called it in the `valid()` function we created above.
+
+<<< ./snippets/App.js.snippet:7[js]
+
+Now, add the following function outside the App component to display the messages:
+
+<<< ./snippets/App.js.snippet:8[js]
+
+## Send messages
+
+We need to write a simple method that will [create](/sdk/js/7/controllers/document/create) a new message document in Kuzzle.
+
+<<< ./snippets/App.js.snippet:9[js]
+
+We will receive notifications from Kuzzle each time we modify our message collection (even if it is a message creation on our part) that we will use to add the messages in our array.
+Now, we need to subscribe to the collection that contains our messages. So let's create our `subscribeMessages()` action. It will call Kuzzle's realtime controller to allow us to [receive notifications](/sdk/js/7/controllers/realtime/subscribe) on message creations:
```

Comment:
```suggestion
Now, we need to subscribe to the collection that contains our messages. So let's create our `subscribeMessages()` method. 
It will call Kuzzle's realtime controller to allow us to [receive notifications](/sdk/js/7/controllers/realtime/subscribe) on message creations:
```